lldb/unittests/Core/TelemetryTest.cpp

```diff
@@ -52,7 +52,7 @@ class FakePlugin : public telemetry::TelemetryManager {
 public:
   FakePlugin()
       : telemetry::TelemetryManager(
-            std::make_unique<llvm::telemetry::Config>(true)) {}
+            std::make_unique<telemetry::LLDBConfig>(true, true)) {}
```

Comment:
It's really hard to tell what both `true` values mean. Can you add inline comments? 
```suggestion
            std::make_unique<telemetry::LLDBConfig>(/*enable_telemetry=*/true, /*detailed_command_telemetry=*/true)) {}
```